api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -24,7 +24,7 @@ option (udpa.annotations.file_status).package_version_status = FROZEN;
 // Redis Proxy :ref:`configuration overview <config_network_filters_redis_proxy>`.
 // [#extension: envoy.filters.network.redis_proxy]
 
-// [#next-free-field: 7]
+// [#next-free-field: 8]
```

Comment:
Since the v2 API is frozen, can you make these changes only to the v3 APIs?

Reply:
Yes I can give that a go today.

Reply:
@htuch The one question I have is around test.   The tests in these two files are based on a v2 config file format:
github.com/envoyproxy/envoy/blob/master/test/extensions/filters/network/redis_proxy/redis_proxy_integration_test.cc
github.com/envoyproxy/envoy/blob/master/test/extensions/clusters/redis/redis_cluster_integration_test.cc

Do these need new tests that are based on a v3 config format or are these tests only relevant to the v2 api?